packages/core/src/tooltip/Tooltip.css

```diff
@@ -1,39 +1,41 @@
-.uitk-density-touch .uitkTooltip {
+.uitk-density-touch {
   --icon-top: 6px;
 }
 
-.uitk-density-low .uitkTooltip {
+.uitk-density-low {
   --icon-top: 4px;
 }
 
-.uitk-density-medium .uitkTooltip {
+.uitk-density-medium {
   --icon-top: 1px;
 }
 
-.uitk-density-high .uitkTooltip {
+.uitk-density-high {
   --icon-top: 1px;
 }
 
+/* Styles applied to root component */
 .uitkTooltip {
-  --tooltip-background: var(--uitk-container-background-medium);
-  --tooltip-font-size: var(--uitk-text-fontSize);
-  --tooltip-color: var(--uitk-text-primary-foreground);
-  --tooltip-padding: var(--uitk-size-unit);
+  --tooltip-background: var(--uitkTooltip-background, var(--uitk-container-background-medium));
+  --tooltip-fontSize: var(--uitkTooltip-fontSize, var(--uitk-text-fontSize));
+  --tooltip-lineHeight: var(--uitkTooltip-lineHeight, var(--uitk-text-lineHeight));
 }
 
 .uitkTooltip {
   background: var(--tooltip-background);
   box-sizing: border-box;
-  border: 1px solid var(--tooltip-status-color);
-  box-shadow: var(--uitk-overlayable-shadow-popout);
-  color: var(--tooltip-color);
-  font-size: var(--tooltip-font-size);
-  line-height: var(--uitk-text-lineHeight);
-  min-height: calc(var(--tooltip-font-size) * var(--uitk-text-lineHeight));
-  padding: var(--tooltip-padding);
+  border-color: var(--uitkTooltip-borderColor, var(--tooltip-status-borderColor));
+  border-width: var(--uitkTooltip-borderWidth, var(--uitk-container-borderWidth));
+  border-style: var(--uitkTooltip-borderStyle, var(--uitk-container-borderStyle));
+  box-shadow: var(--uitkTooltip-shadow, var(--uitk-overlayable-shadow-popout));
+  color: var(--uitkTooltip-text-color, var(--uitk-text-primary-foreground));
+  font-size: var(--tooltip-fontSize);
+  line-height: var(--tooltip-lineHeight);
+  max-width: var(--uitkTooltip-maxWidth, 230px);
+  min-height: calc(var(--tooltip-fontSize) * var(--tooltip-lineHeight));
```

Comment:
not sure if this formula is correct. what if they set '--uitkTooltip-lineHeight' in px?